aws-provision-arm.sh

```diff
@@ -0,0 +1,82 @@
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04/Release.key | sudo apt-key add -
+
+sudo apt-get update -y
+sudo apt-get install -y iproute2 ipset iptables nfs-common ssl-cert libnvidia-container-tools
+#sudo apt-get install -y --no-install-recommends nvidia-cuda-toolkit-10-0 nvidia-driver-430
+
+#sudo apt-get install -y podman
+# Remove parameter not supported by systemd on 16.04.
+#sudo sed -i '/StartLimitIntervalSec/d' /usr/lib/systemd/system/podman.service
+sudo systemctl daemon-reload
+
+curl -L https://toolbelt.treasuredata.com/sh/install-ubuntu-bionic-td-agent3.sh | sh
+sudo sed -i '/^User=.*$/d' /lib/systemd/system/td-agent.service
+sudo sed -i '/^Group=.*$/d' /lib/systemd/system/td-agent.service
+sudo apt-get install -y g++ make
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 0.7.4 fluent-plugin-cloudwatch-logs
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 2.3.0 fluent-plugin-concat
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 2.5.1 fluent-plugin-grok-parser
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 1.0.2 fluent-plugin-json-in-json-2
+#sudo /opt/td-agent/embedded/bin/fluent-gem install --version 2.4.0 fluent-plugin-kubernetes_metadata_filter
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 1.0.0 fluent-plugin-multi-format-parser
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 1.5.0 fluent-plugin-prometheus
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 2.0.1 fluent-plugin-record-modifier
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 2.2.0 fluent-plugin-rewrite-tag-filter
+sudo /opt/td-agent/embedded/bin/fluent-gem install --version 1.0.2 fluent-plugin-systemd
+sudo apt-get remove -y g++ make
+sudo mv /tmp/aws-fluentd-cell.conf /etc/td-agent/td-agent.conf
+# verify fluentd config
+sudo REGION=us-east-1 CLUSTER_NAME=dummy /opt/td-agent/embedded/bin/fluentd --dry-run -c /etc/td-agent/td-agent.conf
+sudo systemctl daemon-reload
+sudo systemctl enable td-agent
+#curl -sfLO https://s3.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb
+#sudo dpkg -i amazon-cloudwatch-agent.deb && rm amazon-cloudwatch-agent.deb
+#sudo cp /tmp/aws-cloudwatch-agent.conf /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json
+
+#sudo systemctl enable podman.socket
+
+sudo dpkg -i /tmp/$KIP_PACKAGE
+
+sudo apt-get remove -y unattended-upgrades
+sudo apt-get autoremove -y
+
+sudo mkdir -p /usr/share/collectd
+sudo touch /usr/share/collectd/types.db
+
+sudo mkdir -p /var/lib/docker/containers
+
+sudo mkdir -p /var/log/journal
+
+sudo rm -rf /root/.ssh
+sudo rm -rf /home/packer/.ssh
+sudo rm -rf /home/ubuntu/.ssh
+
+#podman system info
+
+/usr/bin/itzo --version
+# TODO: add defaultURL and version flag to itzo-launcher, use this flag here
```

Comment:
So this is basically just a copy of `aws-provision.sh` with podman disabled? That could be achieved via using/checking an environment variable instead of creating a copy of the whole script.